src/functions/mod.ts

```diff
@@ -66,3 +106,39 @@ export class SlackFunctionDefinition<
     };
   }
 }
+
+export class ConnectorDefinition<
```

Comment:
Adding a `DefineConnector` function felt redundant, we can just do `new ConnectorFunctionDefinition({})` instead
Lmk what you think

Reply:
Our `Define*` methods instantiating and returning the class instance is our established pattern. Let's stick to it.

This pattern pre-dates me and I never discussed it in depth, but I always assumed it was because functions are a more common pattern in JS libs than classes